data/sponsors.json

```diff
@@ -0,0 +1,33 @@
+{
+    "sponsors":{
```

Comment:
Uh you don't need to have this key once you remove the border key.

Reply:
I will close this PR only the can create another PR